fleetspeak/build-pkgs.sh

```diff
@@ -18,20 +18,23 @@ set -e
 /bin/echo >&2 ""
 /bin/echo >&2 "Building server.deb"
 
+DEB_DEST="server-pkg/debian/fleetspeak-server"
+
 fakeroot bash -c '
   rm -rf server-pkg
   cp -r server-pkg-tmpl server-pkg
 
   chmod 755 server-pkg/*
 
-  sed -i "s/<version>/$(cat ../VERSION)/" server-pkg/DEBIAN/control
+  sed -i "s/<version>/$(cat ../VERSION)/" server-pkg/debian/changelog
 
   mkdir -p server-pkg/usr/bin
   install -o root -g root src/server/server/server server-pkg/usr/bin/fleetspeak-server
   install -o root -g root src/config/fleetspeak_config server-pkg/usr/bin/fleetspeak-config
-  
-  dpkg-deb -b server-pkg server.deb
-  dpkg-deb -c server.deb
+
+  cd server-pkg
```

Comment:
I suggest generating a changelog file here, and deleting the checked in one. From GRR's experience, developers will most likely not keep a comprehensive history of changes (if they do keep the file updated at all). Doing this also allows you to avoid the 'sed' replace for the version.

Reply:
Done. Switched to using debchange.